### ISSUE TITLE
UI design

### ISSUE THREAD

**Issue by Fluoflo** [linagora/llng-manager#20]
Here's a first design suggestion for the Configuration screen. It follows @Eriikah refactor of the UI in  #12 
![llng-manager-configuration-v1](https://github.com/user-attachments/assets/68de2af7-653f-46a2-acad-02d671291293)

What do you think of that ? Did I missed any features/data ?

**Comment by Eriikah** [comment]
I like it, and you didn't miss anything! 
How did you make this view? Did you make a branch?
I'll be sure to include this image in my report, thanks a lot!

**Comment by Fluoflo** [comment]
No, it is not another branch. It is only a mock-up made with Penpot (an Open Source equivalent of Figma) for now.

**Comment by Fluoflo** [comment]
Design suggestion for the application page.

![llng-manager-app_oidc_basic-v1](https://github.com/user-attachments/assets/87626ad4-c2db-4965-a772-8d349fc099ea)
![llng-manager-app_oidc_option_advanced-v1](https://github.com/user-attachments/assets/f80bd59a-a402-4f1d-aa0d-10101e5e51a4)
